boards/arm/nucleo_g474re/doc/index.rst

```diff
@@ -119,6 +119,8 @@ The Zephyr nucleo_g474re board configuration supports the following hardware fea
 +-----------+------------+-------------------------------------+
 | ADC       | on-chip    | adc                                 |
 +-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | rtc                                 |
```

Comment:
Need to also add counter to the supported list in the board yaml for nucleo_g474re and nucleo_g431rb boards

Reply:
nucleo_g474re and nucleo_g431rb have -counter in their _yaml_ and rtc in their _dts_